tests/test_e2e_80_pathfinder.py

```diff
@@ -0,0 +1,313 @@
+import json
+import requests
+from tests.helpers import NetworkTest
+import tests.helpers
+import time
+import pytest
+
+CONTROLLER = "127.0.0.1"
+KYTOS_API = "http://%s:8181/api/kytos" % CONTROLLER
+source = "00:00:00:00:00:00:00:01:3"
```

Comment:
Could you move both `source` and `destination` as `self.source` and `self.destination` on `setup_method`? 

`CONTROLLER` and `KYTOS_API` could also be moved, but since other test suites also use these global vars so it's OK to leave them as it is.